third_party/3.6/click/decorators.pyi

```diff
@@ -29,6 +33,7 @@ def command(
     name: str = None,
     cls: type = Command,
     # Command
+    context_settings: Dict = None,
```

Comment:
:param context_settings: an optional dictionary with defaults that are passed to the context object.

This can be a large Union of things? Do we really want something like:
Optional[Dict[str, Optional[Union[str, bool, int, List[str]]]]

Example use I commonly do:
context_settings = {'help_option_names': ['-h', '--help']}